src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -103,6 +103,11 @@ @implementation GRPCCall {
 
 @synthesize state = _state;
 
+// TODO(jcanizales): If grpc_init is idempotent, this should be changed from load to initialize.
++ (void)load {
```

Comment:
Perhaps put this in the comment, so another person new to ObjC doesn't have this question.